bin/set_display_names.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+#
+# Update the applications display name based on the release env file
+#
+# Usage
+#
+#   bin/set_display_names <?env>
+#
+# Example
+#
+#   bin/fetch_env
+#
+# Requirements
+#
+# 1. An .env.bt.release file in the root folder or an existing environment file
+# in the route of the argument
+
+require 'dotenv'
+require 'open3'
+
+# Constants
+ENV_FILE = ARGV[0] || ".env.bt.release"
+DISPLAY_NAME_KEY = "DISPLAY_NAME"
+PLIST_PATH = "./ios/BT/Info.plist"
+ANDROID_STRINGS_PATH="./android/app/src/bt/res/values/strings.xml"
+SEPARATOR = "##################################################################"
+
+def get_current_ios_name
+  output, error, status = Open3.capture3(
+    "/usr/libexec/PlistBuddy -c \"Print :CFBundleDisplayName\" \"#{PLIST_PATH}\""
+  )
+  if status.success?
+    output.gsub(/\R+/, "")
+  else
+    puts "Could not read ios current name from file #{PLIST_PATH}, error: #{error}"
+    exit 1
+  end
+end
+
+def update_ios_display_name(new_name)
+  puts "Updating ios display name from #{get_current_ios_name} to #{new_name}"
+  output, error, status = Open3.capture3(
+    "/usr/libexec/PlistBuddy -c \"Set :CFBundleDisplayName #{new_name}\" \"#{PLIST_PATH}\""
```

Comment:
Opted for using the local libraries for `macos` happy to add ruby specific libraries, for that we should add a `Gemfile` in the root of the project. This felt lightweight  and the right amount of effort for the task.

Reply:
This makes sense to me.  I dont think we should avoid adding gemfile persay, but keeping the build process as lean as possible is a good goal, imo.